### PR TITLE
Improved help text to fix issue #35, updated setup.py to fix issue #39

### PR DESCRIPTION
**Pull request by AndrewJanuary**

### CODE REVIEW

File: octopose/deploy.py [hunk 1]
```diff
@@ -32,11 +32,14 @@
...
+    parser.add_argument('infile', nargs='?', type=argparse.FileType('rb'), default=sys.stdin,
+                        help='Supply a .json manifest file as an input')
     parser.add_argument('-e', '--environment', default="local", type=str,
                         help="Supply the environment that you would like to deploy to.")
-    parser.add_argument('--wait', action="store_true", help="Wait and poll for deploy")
-    parser.add_argument('--force', action='store_true', help='Force deployment')
+    parser.add_argument('--wait', action="store_true",
+                        help="Wait and poll for deploy. Octopose will continually poll the Octopus Deploy Tasks till they are complete")
+    parser.add_argument('--force', action='store_true',
+                        help='Force deployment. Ensures the package is re-downloaded even if it is already deployed into the target environment')
```
Comment (george-ayris) [inline octopose/deploy.py]: Tiny one, can this say "Ensures the package is re-deployed even if the selected version is already deployed into the target environment"

**Review by domgreen — CHANGES_REQUESTED** [review]
Can you split into 2 seperate PRs one for each issue.

File: octopose/deploy.py [hunk 2]
```diff
@@ -32,11 +32,14 @@
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('infile', nargs='?', type=argparse.FileType('rb'), default=sys.stdin)
+    parser.add_argument('infile', nargs='?', type=argparse.FileType('rb'), default=sys.stdin,
+                        help='Supply a .json manifest file as an input')
     parser.add_argument('-e', '--environment', default="local", type=str,
                         help="Supply the environment that you would like to deploy to.")
```
Comment (domgreen) [inline octopose/deploy.py]: QQ - is `environment` what Octopus show in their UI?

File: octopose/deploy.py [hunk 3]
```diff
@@ -32,11 +32,14 @@
...
-    parser.add_argument('--wait', action="store_true", help="Wait and poll for deploy")
-    parser.add_argument('--force', action='store_true', help='Force deployment')
+    parser.add_argument('--wait', action="store_true",
+                        help="Wait and poll for deploy. Octopose will continually poll the Octopus Deploy Tasks till they are complete")
```
Comment (domgreen) [inline octopose/deploy.py:40]: `complete.`

File: octopose/deploy.py [hunk 4]
```diff
@@ -32,11 +32,14 @@
...
+    parser.add_argument('--force', action='store_true',
+                        help='Force deployment. Ensures the package is re-deployed even if the selected version is already deployed into the target environment')
```
Comment (domgreen) [inline octopose/deploy.py:42]: `environment.` or remove from `-e` 👍

File: octopose/deploy.py [hunk 5]
```diff
@@ -32,11 +32,14 @@
...
     parser.add_argument('-v', '--verbose', action='store_true', help="Do not suppress logging from deploy scripts.")
```
Comment (domgreen) [inline octopose/deploy.py:43]: I would update this message ... verbose says we are getting extra debug info IMO ... up to you guys. But would change to something along the lines of `Verbose logging gives extra contextual output.`
Also, what is the default on this arg?  I get confused with this one ... by default we should not log at verbose level.

**Review by domgreen — APPROVED** [review]
LGTM - just a few nits / queries.

Thanks for contributing would be good to see more step up 🤘

Feel free to keep both issues in here but would be good to split further issues.

File: octopose/deploy.py (hunk 5 above)
Comment (george-ayris) [reply octopose/deploy.py:43]: Default is do not log at the verbose level

**Pull request merged**